.github/workflows/publish-commit-bottles.yml

```diff
@@ -335,11 +335,28 @@ jobs:
 
           echo "head_sha=$(git rev-parse HEAD)" >> "$GITHUB_OUTPUT"
 
-      - name: generate build provenance
-        uses: github-early-access/generate-build-provenance@main
+      - name: Generate build provenance
+        uses: actions/attest-build-provenance@v1
         with:
           subject-path: '${{steps.pr-pull.outputs.bottle_path}}/*.tar.gz'
-  
+
+      - name: Upload bottles to GitHub Packages
+        id: pr-upload
+        working-directory: ${{steps.set-up-homebrew.outputs.repository-path}}
+        env:
+          BREWTESTBOT_NAME_EMAIL: "BrewTestBot <1589480+BrewTestBot@users.noreply.github.com>"
```

Comment:
Probably also need `HOMEBREW_GPG_PASSPHRASE` here as `pr-upload` generates the bottle commit.

Reply:
Thanks! In that case I think `pr-pull` doesn't need that, right?

Reply:
Yeah, and likely don't need committer stuff either since we do `--no-cherry-pick --clean`